src/components/Text.js

```diff
@@ -22,19 +19,14 @@ Knows:
 - Parent block
 
 */
-export default class Text extends mix.withBase( Object3D )(
-    InlineComponent,
-    TextManager,
-    MaterialManager,
-    MeshUIComponent
-) {
```

Comment:
With this old way of mixing things, it is not clear that `MaterialManager` **must** be mixed with `TextManager`, or else `TextManager` will break. This PR fixes this design bug.